src/FxCop/Microsoft.ApiDesignGuidelines.Analyzers/Test/Usage/CollectionPropertiesShouldBeReadOnlyTests.cs

```diff
@@ -0,0 +1,846 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.CodeAnalysis.UnitTests;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace System.Runtime.Analyzers.UnitTests
+{
+    public partial class DisposableFieldsShouldBeDisposedTests : DiagnosticAnalyzerTestBase
```

Comment:
Not done yet ☺


-          heejae

From: Srivatsn Narayanan [mailto:notifications@github.com]
Sent: Friday, November 6, 2015 8:54 AM
To: dotnet/roslyn-analyzers <roslyn-analyzers@noreply.github.com>
Cc: HeeJae Chang <Heejae.Chang@microsoft.com>
Subject: Re: [roslyn-analyzers] Ported CA2227 (#345)


In src/FxCop/Microsoft.ApiDesignGuidelines.Analyzers/Test/Usage/CollectionPropertiesShouldBeReadOnlyTests.cs<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fdotnet%2froslyn-analyzers%2fpull%2f345%23discussion_r44159461&data=01%7c01%7cHeejae.Chang%40microsoft.com%7c55dac4ff547146aa3f0b08d2e6cad9c1%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=qYgpC3TJCKP5N0zGOD%2fas0qrjxTMQlLxYdjBYBNT%2f0c%3d>:

> @@ -0,0 +1,846 @@

> +﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.

> +

> +using Microsoft.CodeAnalysis.Diagnostics;

> +using Microsoft.CodeAnalysis.Test.Utilities;

> +using Microsoft.CodeAnalysis.UnitTests;

> +using Roslyn.Test.Utilities;

> +using Xunit;

> +

> +namespace System.Runtime.Analyzers.UnitTests

> +{

> +    public partial class DisposableFieldsShouldBeDisposedTests : DiagnosticAnalyzerTestBase

What happened here? These are tests for a different rule

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fdotnet%2froslyn-analyzers%2fpull%2f345%2ffiles%23r44159461&data=01%7c01%7cHeejae.Chang%40microsoft.com%7c55dac4ff547146aa3f0b08d2e6cad9c1%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=ETyUHknQj0w0e19W%2bSdkffhR3J968pmX0tUBYq%2bYy84%3d>.